### PR TITLE
Fix #6508: Add config to enable gpu's for docker agents

### LINKED ISSUE #6508: job_driver.docker needs to support gpus
Need to be able to set `--gpus=all`

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: sirepo/job_driver/docker.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def init_class(cls, job_supervisor):
                 bool,
                 "mount ~/.pyenv, ~/.local and ~/src for development",
             ),
+            enable_gpus=(False, bool, "apply --gpus=all"),
```
Comment (gurhar1133) [inline sirepo/job_driver/docker.py:96]: `"apply '--gpus=all'"`? Clearer?

**Review by gurhar1133 — APPROVED** [review]
Left one comment (might not really matter). Approve assuming that tests pass

File: sirepo/job_driver/docker.py (hunk 1 above)
Comment (robnagler) [reply sirepo/job_driver/docker.py:96]: Make this identical to rsdockerspawner, which is what docker cli expects.

Comment (e-carlin) [reply sirepo/job_driver/docker.py:96]: @robnagler can you elaborate on what you'd like to see? [This is what I see](https://github.com/radiasoft/rsdockerspawner/blob/6783877fa9bb526a9da06e3cd6e4700d80e0e618/rsdockerspawner/rsdockerspawner.py#L105) in rsdockerspawner. When you set `--gpus=all` on the command line (as is done here) it sets the DeviceRequests config that rsdockerspawner sets itself.

Comment (e-carlin) [reply sirepo/job_driver/docker.py]: Thanks b0cbf7352

**Pull request merged**